e2e/tests/ui-driven/src/helpers/userActions.ts

```diff
@@ -194,13 +193,6 @@ export async function submitCardDetails(page: Page) {
   await page.locator("#confirm").click();
 }
 
-export async function answerFindProperty(page: Page) {
```

Comment:
Moved this to a separate `geoSpatialUserActions` doc to fit it better with the other ones